ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -159,15 +164,20 @@ private void SetSetting<T>(string key, T value)
             var values = ApplicationData.Current.LocalSettings.Values;
             values[key] = value;
 #else
-            var values = ConfigurationManager.AppSettings;
-            if (values[key] == null)
+            var configFile = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+            var settings = configFile.AppSettings.Settings;
```

Comment:
I have implemented a simple Dictionary to store these values in memory... The settings will not be persisted across application instances...